src/features/mcda/components/MCDAForm/index.tsx

```diff
@@ -14,7 +14,12 @@ import { availableBivariateAxisesAtom } from '../../atoms/availableBivariateAxis
 import s from './style.module.css';
 import type { Axis } from '~utils/bivariate';
 
-type FormState = {
+type FormInitialState = {
+  name: string;
+  axises: { id: string; label: string }[];
```

Comment:
axes

Reply:
Yeah, I know it's axes in English, I decided to keep it consistent with the rest of the codebase, where this form is used. That would be too much refactoring to change everywhere

Reply:
https://kontur.fibery.io/Tasks/Task/Change-typo-in-codebase-axises---axes-18017

Reply:
@Komzpa thanks!